src/main/java/hudson/plugins/jira/JiraChangeLogAnnotator.java

```diff
@@ -26,13 +27,36 @@
  */
 @Extension
 public class JiraChangeLogAnnotator extends ChangeLogAnnotator {
+	
     private static final Logger LOGGER = Logger.getLogger(JiraChangeLogAnnotator.class.getName());
 
+    public JiraChangeLogAnnotator()
+    {
+        super();
+        if(LOGGER.isLoggable(Level.FINE))
+        {
+        	LOGGER.info("JiraChangeLogAnnotator registered");
+        }
+    }
+    
     @Override
-    public void annotate(AbstractBuild<?, ?> build, Entry change, MarkupText text) {
-        JiraSite site = getSiteForProject(build.getProject());
-        if (site == null) return;    // not configured with JIRA
-
+    public void annotate(Run<?, ?> build, Entry change, MarkupText text) {
+        JiraSite site = getSiteForProject(build.getParent());
+        
+        if (site == null)
+    	{
+        	if(LOGGER.isLoggable(Level.FINE))
+            {
+        		LOGGER.fine("not configured with JIRA");
+            }
+        	return;    // not configured with JIRA
+    	}
+        
+        if(LOGGER.isLoggable(Level.FINE))
+        {
+        	LOGGER.fine("Using site: " + site.url);
```

Comment:
logger should check by itself the level in this form, so no need to `if(LOGGER.isLoggable(Level.FINE))`